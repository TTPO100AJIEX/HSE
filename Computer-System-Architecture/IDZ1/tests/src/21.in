100
22 -14 30 -6 25 22 8 7 17 23 -10 -12 4 -22 10 6 -19 -1 -29 -19 -5 0 10 27 -20 -11 5 5 6 28 -32 30 1 -2 -9 15 6 -11 16 0 25 -1 -21 2 2 -12 13 -31 5 24 22 -3 6 -32 32 -17 -30 -8 -31 19 24 -8 7 28 -28 -32 -5 -13 27 -31 26 16 -22 -3 -12 -16 22 -1 -2 -32 23 17 9 22 9 13 -25 -6 -29 -31 4 12 -6 -3 14 -6 0 -12 15 -9