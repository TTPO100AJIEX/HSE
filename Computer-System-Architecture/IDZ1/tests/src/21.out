-14 30 -6 25 7 17 23 -12 4 -22 10 -19 -1 -29 -19 -5 0 10 27 -20 -11 5 5 6 28 -32 30 -9 15 -11 16 0 25 -21 2 2 -12 13 -31 5 24 -3 6 -32 32 -30 -8 -31 19 24 -8 7 28 -32 -5 -13 27 -31 26 -22 -3 -16 22 -32 23 9 22 9 13 -25 -6 -31 4 12 -6 -3 14 -6 0 -12 15 